src/tags/page-getters.ts

```diff
@@ -66,19 +64,21 @@ export const getChildTags = ($tagPage: TagPage) => {
 }
 
 export const getSubTags = ($tagPage: TagPage) => {
-  const subTags: { tagName: string; parentSubTag: string | null }[] = [];
-  $tagPage(".sub > ul.tags > li").each((_, element) => {
-    subTags.push({ tagName: $tagPage(element).children().first().text(), parentSubTag: null });
-    if ($tagPage($tagPage(element)).has("ul.tags").length) {
-      $tagPage("ul.tags", element).children("li").each((_, child) => {
-        // each <li> element contains an <a> element, 
-        // which is why `.children().first()` is needed for both the `tagName` and `parentSubTag`
-        subTags.push({ 
-          tagName: $tagPage(child).children().first().text(), 
-          parentSubTag: $tagPage($tagPage(child)).parents("li").children().first().text() 
-        });
-      });
+  return $tagPage(".sub > ul.tags > li").map((_, element) => {
+    if($tagPage($tagPage(element).has("ul.tags")).length) {
```

Comment:
Change this for an early if return (the thing we discussed on Discord) if you can :)